### DISCUSSION TITLE
How do I get the slider min value to work?

### DISCUSSION THREAD

**Discussion by Ian-Bright** [radix-ui/primitives#2499] [Help]
I'm not understanding how the min and max value for a slider range are set. When I set these numbers as props they seem to be ignored and you can drag past the max and min range. What am I doing wrong here?

**Comment by benoitgrelard** [comment]
What do you mean by this? `min` and `max` simply represent the left and right boundaries so you can't drag a value lower or higher than that. This is the same behaviour as the native input range.
